LinuxWADE.R

```diff
@@ -0,0 +1,32 @@
+##################################################################
+####  WGS Analysis and Detection of Molecular Markers (WADE)  ####
+####       Authors: Walter Demczuk & Shelley Peterson         ####
+####                    Date: 2024-04-24                      ####
+##################################################################
+
+library(plyr)
+library(tidyverse)
+library(tidyselect)
+library(tidysq)
+library(Biostrings)
+library(shiny)
+library(shinyWidgets)
+library(DT)
+library(readxl)
+library(beepr)
+library(WADE)  
+
+####### <<<< Change this path to where the WADE directory is stored >>>> #######
+curr_work_dir <- "~/Documents/WADE/"
+#-------------------------------------------------------------------------------
+
+removefiles(curr_work_dir)
```

Comment:
Would this line remove all files from the curr_work_dir every time wade is started up? Or is this script only run once when setting everything up?

Do you need to remove files here? I am only pointing it out since it could lead to unexpected data deletion if someone types in a path above and doesn't realize what is going to happen (or leave curr_work_dir as default `~/Documents/WADE` and had some other data in that directory).

If it is needed to clear out the directory, I'm wondering if `curr_work_dir` should default to `null` then, and an if statement could be added here to only clear out that directory/continue running wade if it's not null, which would force someone to have to change that variable (and know that the directory will be cleared) before running wade.

Reply:
removefiles() is a function specific to WADE that removes specific WADE output files that are set to append rather than replace data (eg. fasta files for the queried sequences) so they're empty before WADE is run and don't become infinitely long. It won't affect any of the other files in the folder. See CommonFunctions.R for more details.

Reply:
Ah, okay. Thanks for the explanation. Could you add a comment above this line so that anyone else looking at the code knows what the function does?

Reply:
ok. I've added the comment

Reply:
Done!

Reply:
Thanks so much 😄 . You can merge in.